src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -1,12 +1,28 @@
 ﻿using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using System.Collections.Generic;
+using System.IO;
 
 namespace Stryker.Core.Mutants
 {
     public static class MutantPlacer
     {
+        private static readonly string helper;
+
+        static MutantPlacer()
+        {
+            using (var stream =
+                typeof(MutantPlacer).Assembly.GetManifestResourceStream("Stryker.Core.Mutants.ActiveMutantHelper.cs"))
```

Comment:
Would something similar to this:

```
Type activeMutantHelper = typeof(ActiveMutantHelper);
using (var stream = activeMutantHelper.Assembly.GetManifestResourceStream(activeMutantHelper.FullName))
```

perhaps be possible so we do not need hardcoded resource file names? I'm pretty sure this example is not complete but perhaps there is a way to use this to format the complete name.

Reply:
Resource file name are hardcoded identifier as those are external resources (from a source code point of view). 
I am not able to implement what you propose as the source code is not built as part of the project, only embedded. Hence there is no type to refer to.

Reply:
Fair enough